test_autoflake.py

```diff
@@ -1224,6 +1224,43 @@ def test_exclude(self):
         finally:
             shutil.rmtree(temp_directory)
 
+    def test_with_ignore_init_module_imports_flag(self):
+        # Need a temp directory in order to specify file name as __init__.py
+        temp_directory = tempfile.mkdtemp(dir='.')
+        try:
+            with open(
+                 os.path.join(temp_directory, '__init__.py'), 'w') as output:
+                output.write('import re\n')
+
+            p = subprocess.Popen(
+                list(AUTOFLAKE_COMMAND) +
+                [temp_directory + '/__init__.py',
+                    '--ignore-init-module-imports'],
```

Comment:
put the `'--ignore-init-module-imports'` before the filename , as that is the more usual way commands are written, and will make the test logic slighly more intuitive for the reader.

Reply:
Fixed!